src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.ThreadCounts.cs

```diff
@@ -16,15 +16,14 @@ private struct ThreadCounts
             // SOS's ThreadPool command depends on this layout
             private const byte NumProcessingWorkShift = 0;
             private const byte NumExistingThreadsShift = 16;
-            private const byte NumThreadsGoalShift = 32;
 
-            private ulong _data; // SOS's ThreadPool command depends on this name
+            private uint _data; // SOS's ThreadPool command depends on this name
```

Comment:
so `ulong` was unnecessary to start with?

Reply:
The `ulong` is left over from the native implementation. The native implementation had another value in the `ThreadCounts` that was intentionally excluded during the port. The only other thing left, the goal, doesn't need to be in the counts and it simplifies the code.